CrewChiefV4/UserInterface/ListPropertyControl.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace CrewChiefV4
+{
+    /// <summary>
+    /// this class holds the list box label -> invariant value mappings for each listbox property used in the properties
+    /// form. 
+    /// </summary>
+    public class ListPropertyValues
+    {
+        static ListPropertyValues()
+        {
+            // listbox items for interrupt property
+            ListBoxItem[] interruptItems = new ListBoxItem[]{
+                new ListBoxItem(Configuration.getUIString("interrupts_none"), "NONE"),
```

Comment:
this can be generalized, by using postfixes for prop_name, see my Discord suggesiton :)